### PR TITLE
reference image upload tests and UX polish (SCP-3758)

### PR DESCRIPTION
**Pull request by devonbush**
This fixes a couple UX papercuts in reference image upload, and adds a test

TO TEST:
 1. open a study in the new upload wizard, and go to the reference image tab
 2. select an image to upload
 3. confirm the 'name' text field displays correctly, and is prepopulated with the filename
 4. upload the image
 5. confirm that a broken image tag does not appear while the file is in 'waiting for remote image' state.
 6. once the image is sent to firecloud, confirm the image is shown

### CODE REVIEW

**Review by bistline — CHANGES_REQUESTED** [review]
I'm not sure where the error is arising, but I was not able to get any images to render in the "Images" tab.  I uploaded images, confirmed they saved and pushed to the bucket, but the image links are broken.  In the JS console, I see these errors for all images:

```
GET https://storage.googleapis.com/download/storage/v1/b/undefined/o/undefined?alt=media 403
```

The bucket name and filename are not being populated correctly.

**Review by bistline — APPROVED** [review]

File: test/js/upload-wizard/upload-images.test.js
Comment (eweitz) [inline test/js/upload-wizard/upload-images.test.js:17]: This copy/paste mistake will confuse maintainers.  Possible fix:

```suggestion
  it('validates bad file names, starts upload of JPEG file', async () => {
```

**Review by eweitz — APPROVED** [review]
Code looks good, overall!  I approve, assuming the trivial copy/paste mistake gets fixed.  

The other comments are nits, and a question.

**Comment by devonbush** [comment]
@eweitz  re: the different names, was the result of a series of merge conflicts and resolutions.  Agree it should be fixed, will tackle it along with the next set of wizard tests

**Pull request merged**
